### ISSUE TITLE
Difficulty with IE.

### ISSUE THREAD

**Issue by nathanhokanson** [running-coder/jquery-typeahead#148]
I am importing the jquery.typeahead via requirejs in the following manner:

``` javascript
var require = {
  shim: {
    'jquery' : {
      exports: "window.jQuery"
    },
    .....
            'libraries/jquery.typeahead.min' : {
            deps:['jquery'],
            init: function($) {
              return require.s.contexts._.registry['jquery.typeahead.min'].factory($);
            }
        }
    },
    paths: {
       'jquery': "libraries/jquery-1.10.2",
        'libraries/jquery.typeahead.min':'libraries/jquery.typeahead.min'
```

**Comment by running-coder** [comment]
hey nathan, I'm not familiar with all requireJs subtilities but mine looks like:

``` javascript
var require = {
    waitSeconds: 0,
    paths: {

        'jquery': '../plugin/jquery/dist/jquery.min',
        'bootstrap': '../plugin/bootstrap/dist/js/bootstrap.min',

        'angular': '../plugin/angular/angular.min',
        'jquery-typeahead': '../plugin/jquery-typeahead/dist/jquery.typeahead.min'

    },
    shim: {
        'bootstrap': {deps: ['jquery']},

        'angular': {'exports': 'angular', deps: ['jquery']},
        'jquery-typeahead': {deps: ['jquery']}
    },
    priority: [
        'jquery',
        'bootstrap',
        'angular'
    ]
};
```

What is the issue / thrown error?

**Comment by nathanhokanson** [comment]
Sorry, the post is incomplete.  Let me try again:

The thrown error is:  SCRIPT438: Object doesn't support property or method 'typeahead'

The definition of my file that uses the typeahead is:

define(['libraries/jquery.typeahead.min'
    ],
    function(TypeAhead){
      // Down in the code I am calling:
      $.typeahead({input:"#templateName", .... });
      // I have also tried $("#templateName").typeahead({ .... })
     }
}

The error occurs at the line where I actually call .typeahead.

Sometimes it succeeds when I reload the page, but also sometimes not.
It works fine in Chrome and Firefox.  I hate IE.

**Comment by running-coder** [comment]
I was able to reproduce the issue using jquery 2.x+ as a dependency, you should either use 1.x or add the jquery migrate file

**Comment by nathanhokanson** [comment]
We are using jquery 1.10.2.

**Comment by running-coder** [comment]
I can see I'm missing 

``` javascript
// At the end of the file before the closing brackets
return $.proxy($.typeahead, Typeahead);
```

**Comment by running-coder** [comment]
let me know if it solved your issue, instead of ``$.typeahead`` you should use the returned param to initialize your typeahead

``` javascript
// For some reason my shim is not working, I still have to include "jquery"...
define(["jquery", "jquery-typeahead"], function($, Typeahead) {

    Typeahead({input:"#templateName", .... })
});
```

**Comment by nathanhokanson** [comment]
Whew!  That fixed the problem!  Thanks!

When will this version be available in a minified format?

**Comment by running-coder** [comment]
Cool! the .min should be committed inside the /dist folder inside the develop branch, https://github.com/running-coder/jquery-typeahead/tree/develop/dist

version 2.3.4 should be available by the end of the week, I still have to squeeze in a couple things

**Comment by nathanhokanson** [comment]
Thanks again for this plugin and all your hard work!

**Comment by running-coder** [comment]
yw good sir, thanks for raising the issue!

**Comment by running-coder** [comment]
I've made a mistake... please update your code when 2.4.0 is released

``` javascript
define(["jquery-typeahead"], function(Typeahead) {

    $.typeahead({input:"#templateName", .... })
});
```
